5-analysis/analysis-powder-diffraction.ipynb

```diff
@@ -0,0 +1,1665 @@
+    "the scale factor.\n",
+    "\n",
+    "**Solution:**\n",
+    "\n",
+    "Before optimizing the parameters, we can visualize the measured\n",
+    "diffraction pattern and the calculated diffraction pattern based on the\n",
+    "two phases: LBCO and Si.\n",
+    "\n",
+    "**Visualize Diffraction Patterns**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "id": "8e6bc18e",
+   "metadata": {},
+   "source": [
+    "project_2.plot_meas_vs_calc(expt_name=\"sim_lbco\")"
+   ],
+   "outputs": [],
+   "execution_count": null
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2a01d9fd",
+   "metadata": {},
+   "source": [
+    "As you can see, the calculated pattern is now the sum of both phases,\n",
+    "and Si peaks are visible in the calculated pattern. However, their intensities\n",
+    "are much too high. Therefore, we need to refine the scale factor of the Si phase.\n",
+    "\n",
+    "**Set Fit Parameters**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "id": "0a683b74",
+   "metadata": {},
+   "source": [
+    "project_2.experiments[\"sim_lbco\"].linked_phases[\"si\"].scale.free = True"
+   ],
+   "outputs": [],
+   "execution_count": null
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c7596964",
+   "metadata": {},
+   "source": [
+    "**Run Fitting**\n",
+    "\n",
+    "Now we can perform the fit with both phases included."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "id": "a7216729",
+   "metadata": {},
+   "source": [
+    "project_2.analysis.fit()"
+   ],
+   "outputs": [],
+   "execution_count": null
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b62290c6",
+   "metadata": {},
+   "source": [
+    "**Visualize Fit Results**\n",
+    "\n",
+    "Let's plot the measured diffraction pattern and the calculated diffraction\n",
+    "pattern both for the full range and for a zoomed-in region around the previously unexplained\n",
+    "peak near 95,000 μs. The calculated pattern will be the sum of the two phases."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "id": "38004527",
+   "metadata": {},
+   "source": [
+    "project_2.plot_meas_vs_calc(expt_name=\"sim_lbco\")"
+   ],
+   "outputs": [],
+   "execution_count": null
+  },
+  {
+   "cell_type": "code",
+   "id": "6fc93114",
+   "metadata": {},
+   "source": [
+    "project_2.plot_meas_vs_calc(expt_name=\"sim_lbco\", x_min=88000, x_max=101000)"
+   ],
+   "outputs": [],
+   "execution_count": null
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d9315e63",
+   "metadata": {},
+   "source": [
+    "All previously unexplained peaks are now accounted for in the pattern, and the fit is improved.\n",
+    "Some discrepancies in the peak intensities remain, but\n",
+    "further improvements would require more advanced data reduction and analysis,\n",
+    "which are beyond the scope of this tutorial."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5df41737",
+   "metadata": {},
+   "source": [
+    "## 📑 Summary\n",
```

Comment:
I like this section a lot. However, we said we wanted to do retrospective sessions at the end of each day. The students shouldn't simply read out what it says here but think about what they did. So does it make sense to omit this section from the notebooks that we give to them?

Reply:
Same as in the other tracks. We do one technique at a time. We discussed that in our last meeting. But I guess not everyone was there and we didn't communicate it.

The idea was to start with one technique and grab the corresponding students and go into a separate room or a corner of the room and do the retrospective. The other students continue to work on their exercises. Then the next technique and then the last. So some students lose a bit of time compared to others because their retrospective is earlier. To make this fairer, we wanted to rotate the order of techniques every day.